public/css/themes/Minimal.css

```diff
@@ -1,140 +1,252 @@
-/* 2025-10-04T13-07-00 v1.3.0 */
-/* changed bg, font, albm-art shdw, no animations */
-/* cuztomized the portrait-mode lyrics tab thing */
+/* Minimal by Semicolonhope */
```

Comment:
No need for this, its automatically generated when your theme shows up in the extension
```suggestion
```